src/test/rustdoc/process-termination.rs

```diff
@@ -0,0 +1,22 @@
+/// A check of using various process termination strategies
```

Comment:
To actually run any of the doctests in this file it needs to contain `// compile-flags:--test`.

Reply:
Oops.